libs/utils/utils.js

```diff
@@ -1052,6 +1052,10 @@ async function checkForPageMods() {
 
 async function loadPostLCP(config) {
   await decoratePlaceholders(document.body.querySelector('header'), config);
+  const sk = document.querySelector('helix-sidekick');
```

Comment:
Should we move this into `initSidekick` and move the init sidekick method further up to be called even earlier than the LCP? 

If authors/devs/stakeholders have the sidekick open, the CWV measures are distorted anyway also in regards to say CLS.

Reply:
Tbh, I'd be down. I'll move it 🙌